<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPassenger</class>
 <widget class="QDialog" name="AddPassenger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(251,102,102, 200), stop:1 rgba(20,196,188, 210));</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="7">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 50px;
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="lineEditAge">
        <property name="styleSheet">
         <string notr="true">font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;
height: 30px;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 290 15pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(255, 255, 255);
font: 25 14pt '微软雅黑 Light';</string>
        </property>
        <property name="text">
         <string>年龄</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 290 15pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(255, 255, 255);
font: 25 14pt '微软雅黑 Light';</string>
        </property>
        <property name="text">
         <string>身份证号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditName">
        <property name="styleSheet">
         <string notr="true">font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;
height: 30px;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEditPhone">
        <property name="styleSheet">
         <string notr="true">font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;
height: 30px;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QComboBox" name="comboBoxGender">
        <property name="styleSheet">
         <string notr="true">QComboBox {
    font: 25 14pt '微软雅黑 Light';
    color: rgb(31,31,31);
    padding-left: 20px;
    padding-right: 20px; /* Applied padding-right to the QComboBox */
    background-color: rgb(255, 255, 255);
    border: 2px solid rgb(20,196,188);
    border-radius: 15px;
    height: 30px;
    selection-background-color: darkblue;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: center right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
    background: transparent; /* 确保箭头背景透明 */
}

QComboBox::down-arrow {
    image: none; /* 移除默认箭头图像 */
    border: solid black; /* 设置三角形边框的颜色 */
    border-width: 0 1px 1px 0; /* 设置三角形边框的宽度 */
    width: 8px; /* 设置三角形箭头的宽度 */
    height: 8px; /* 设置三角形箭头的高度 */
    margin: 5px; /* 设置三角形箭头的外边距 */
    -qt-block-indent: 0; /* This is the correct property for rotation in Qt CSS */
    -qt-arrow-indent: 1px; /* This helps with the rotation and positioning of the arrow */
}

QComboBox QAbstractItemView {
    /* 设置下拉列表视图的样式 */
    border: 1px solid darkgray;
    selection-background-color: rgba(20,196,188, 210);
    background-color: rgba(255,255,255, 255);
}
</string>
        </property>
        <item>
         <property name="text">
          <string>男</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>女</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEditID">
        <property name="styleSheet">
         <string notr="true">font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;
height: 30px;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 290 15pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(255, 255, 255);
font: 25 14pt '微软雅黑 Light';</string>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 290 15pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(255, 255, 255);
font: 25 14pt '微软雅黑 Light';</string>
        </property>
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 290 15pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(255, 255, 255);
font: 25 14pt '微软雅黑 Light';</string>
        </property>
        <property name="text">
         <string>电话号码</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QLabel" name="note">
        <property name="styleSheet">
         <string notr="true">font: 290 15pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 85, 127);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="styleSheet">
      <string notr="true">QPushButton {
height: 30px;
font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;

}
QPushButton:hover {
    background-color: rgb(22,218,208);
}
QPushButton:pressed {
    background-color: rgb(17,171,164);
}</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonSave">
     <property name="styleSheet">
      <string notr="true">QPushButton {
height: 30px;
font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;

}
QPushButton:hover {
    background-color: rgb(22,218,208);
}
QPushButton:pressed {
    background-color: rgb(17,171,164);
}</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="8">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
